support-utils/cre_astro_software_gather_logs.sh

```diff
@@ -0,0 +1,392 @@
+#!/bin/bash
+timestamp=`date '+%d/%m/%Y %H:%M:%S'`
+# Put the name of the astronomer ASTRONOMER_NAMESPACE below
+#export ASTRONOMER_NAMESPACE=astronomer
+#export DIR="/tmp"
+#export BASEDOMAIN=nandlal51.astro-cre.com 
+echo "****Example to Run This Script:****
+
+Enter your Astronomer Namespace Name: (This is the namespace where all core Astronomer components like Houston, Commander, Registry, etc., are running)
+astronomer <============ 1
+Enter the directory path where you want to save your exported log files:
+/tmp     <============ 2
+If you have a test cluster with the URL https://app.cre-software-01.astro-cre.com, then your base domain would be cre-software-01.astro-cre.com.
+Enter your base domain:
+cre-software-01.astro-cre.com  <============ 3
```

Comment:
This also needs to be replaced with example.com https://www.rfc-editor.org/rfc/rfc2606